hyperspy/samfire_utils/samfire_pool.py

```diff
@@ -51,8 +51,8 @@ class SamfirePool(ParallelPool):
         given SAMFire object, populates the workers with the required
         information. In case of multiprocessing, starts worker listening to the
         queues.
-    update_optional_names
-        updates the optional names in all workers
+    update_parameters
```

Comment:
Huh, true.... 
On the other hand, I doubt anyone else was aware of the function. On the yet another hand, this "bug" is actually __very__ bad if you do anything else that what I was used to - not only you can't change the parameter boundaries, but also the GOF test, other things... Your call anyway. I can PR to the major branch just as well